service/local-dev/run_server.sh

```diff
@@ -88,13 +88,8 @@ if [[ ${disableAuthChecks} ]]; then
 else
   echo "Enabling auth checks. bearer-token"
   export TANAGRA_AUTH_DISABLE_CHECKS=false
-  if [[ ${useAouUnderlays} ]]; then
-    export TANAGRA_AUTH_BEARER_TOKEN=true
-    export TANAGRA_AUTH_IAP_GKE_JWT=false
-  elif [[ ${useSdUnderlays} ]]; then
-    export TANAGRA_AUTH_BEARER_TOKEN=false
-    export TANAGRA_AUTH_IAP_GKE_JWT=true
-  fi
+  export TANAGRA_AUTH_BEARER_TOKEN=true
```

Comment:
What is this change for?

Reply:
I think the intent here was to allow disabling auth checks or enabling them with the Google access bearer token configuration. The disable part of the conditional is correct (sets `TANAGRA_AUTH_DISABLE_CHECKS=true`), but this enable part was not always using the bearer token method. For SD underlays, it was acting differently -- expecting an IAP token -- which I don't think really makes sense for a locally running service.